app/overrides/spree/admin/variants/_form/on_demand_script.html.haml.deface

```diff
@@ -0,0 +1,28 @@
+/ insert_bottom "[data-hook='admin_variant_form_fields']"
+:javascript
```

Comment:
I'm not overly fussed about it, but seeing as we're doing everything else in coffeescript, might be worth doing this as :coffee instead?